database/src/test/java/org/trellisldp/ext/microprofile/database/PgsqlExtension.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.trellisldp.ext.microprofile.database;
+
+import static org.eclipse.microprofile.config.ConfigProvider.getConfig;
+
+import com.opentable.db.postgres.embedded.EmbeddedPostgres;
+
+import org.apache.commons.text.RandomStringGenerator;
+import org.junit.jupiter.api.extension.Extension;
+
+final class PgsqlExtension implements Extension {
```

Comment:
This is nice.

Reply:
I tend not to reuse the classes in ./src/test (and this is my very first JUnit5 extension). Plus, there really isn't much code here, but I do see where you're going here. I'd probably just end up copying this code into the other project.